src/mergeResourcesAsInterface.js

```diff
@@ -1,11 +1,85 @@
 import mergeResources from './mergeResources.js'
 
-function mergeResourcesAsInterface (namespaces) {
-  const resources = mergeResources(namespaces)
+function mergeResourcesAsInterface(namespaces, options = {}) {
+  let resources = mergeResources(namespaces)
   let interfaceFileContent = 'interface Resources '
+
+  if (options.optimize) {
+    resources = groupPluralKeys(Object.entries(resources), options)
+  }
+  console.dir(resources, {depth: null})
+
   interfaceFileContent += JSON.stringify(resources, null, 2)
   interfaceFileContent += '\n\nexport default Resources;\n'
   return interfaceFileContent
 }
 
+const isPluralKey = (key, {pluralSeparator = defaultOptions.pluralSeparator}) => pluralSuffixes
+  .map((suffix) => `${pluralSeparator}${suffix}`)
+  .some((suffix) => key.endsWith(suffix))
+
+export const defaultOptions = {
+  pluralSeparator: '_'
+}
+
+export const pluralSuffixes = [
+  'zero',
+  'one',
+  'two',
+  'few',
+  'many',
+  'other'
+]
+
+const isPluralEntry = (options) => (entry) => isPluralKey(entry[0], options)
+
+export function rmPluralization(entry, options) {
+  if (!isPluralEntry(options)(entry)) return null
```

Comment:
Continuing suggestion from above:

```suggestion
  if (!isPluralEntry(entry, options)) return null
```